.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,31 @@
+## Pull Request Type
+<!-- Please select which type of change this most aligns with. If more than one type fits, please select multiple. -->
+- [ ] GitHub Workflow changes
+- [ ] Documentation or Wiki changes
+- [ ] Build changes
+- [ ] Dependency changes
```

Comment:
That can be merged into one point Build / dependency changes

Reply:
I was thinking of existing pulls when I wrote it, like the libacm edits that are library changes and not to the game itself.